organizations/user.go

```diff
@@ -171,3 +171,41 @@ func UpdateProfilePicture(w http.ResponseWriter, r *http.Request) {
 	utils.GetSuccess("Profile picture updated", result, w)
 
 }
+
+// Delete single member from an organizatin
+func DeleteMember(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	member_collection, org_collection := "members", "organizations"
+	orgId := mux.Vars(r)["id"]
+	memberId := mux.Vars(r)["mem_id"]
+
+	pOrgId, err := primitive.ObjectIDFromHex(orgId)
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	// get organization
+	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": pOrgId})
+	if orgDoc == nil {
+		fmt.Printf("org with id %s doesn't exist!", orgId)
+		utils.GetError(errors.New("operation failed"), http.StatusBadRequest, w)
+		return
+	}
+
+	query, _ := primitive.ObjectIDFromHex(memberId)
+	checkMember, err := utils.GetMongoDbDoc(member_collection, bson.M{"_id": query})
```

Comment:
There may not be error in this case so err will be nil. Lets have something like this.

user, _ := utils.GetMongoDbDoc(user_collection, bson.M{"_id": objId})
	if user == nil {
		fmt.Printf("users with id %s exists!", newOrg.CreatorID)
		utils.GetError(errors.New("operation failed"), http.StatusBadRequest, w)
		return
	}

We are not interested in the error response so it is replaced with _, then we create our own custom error with the errors.new when our doc is empty